cognite_toolkit/_cdf_tk/loaders/_resource_loaders/location_loaders.py

```diff
@@ -56,6 +59,22 @@ class LocationFilterLoader(
 
     subfilter_names = ("assets", "events", "files", "timeseries", "sequences")
 
+    def __init__(self, client: ToolkitClient, build_dir: Path | None, console: Console | None = None) -> None:
+        self.client = client
+        self.resource_build_path: Path | None = None
+
+        # location filters may have to be deployed in a specific order since the API doesn't
+        # support parentExternalId so we need to deploy them one by one
+        if build_dir and build_dir.is_file():
+            self.resource_build_path = build_dir
```

Comment:
I don't understand this modification of the resource_build_path. It should never be a file always a directory